forte/data/caster.py

```diff
@@ -78,3 +81,40 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This pass on a single DataPack within the MultiPack.
+    """
+
+    def cast(self, pack: MultiPack, pack_index: int = 0) -> DataPack:
+        """
+        Auto-box the MultiPack into a DataPack by using pack_index to take the unique pack.
+
+        Args:
+            pack: The MultiPack to be boxed.
+
+        Returns: A DataPack boxed from the MultiPack.
+
+        """
+
+        if pack_index in range(pack.num_pack):
```

Comment:
to check if the index is in range, I think we can simply check for less?

`if pack_index < range(pack.num_pack)`